### ISSUE TITLE
translations with invalid timings

### ISSUE THREAD

**Issue by alexdean** [pculture/unisubs#794]
https://www.amara.org/api2/ted/videos/qRY6b7syc8E/languages/es/subtitles/?format=ted
```json
    {
      "content": "Entonces llega a la escuelay a una hora de estar en la escuela",
      "duration": 2170,
      "startOfParagraph": false,
      "startTime": 1079229
    }
...
    {
      "content": "[Señoras de almuerzo:] \"¿Quieres fruta hoy?\"",
      "duration": 2170,
      "startOfParagraph": false,
      "startTime": 1079229
    }
```

The elements in the `captions` array also appear out of time order. This isn't critical, but seems unusual.

---

https://www.amara.org/api2/ted/videos/x-_kr7kwCM4/languages/es/subtitles/?format=ted

```json
    {
      "content": "(Aplausos)",
      "duration": 0,
      "startOfParagraph": false,
      "startTime": 475332
    },
    {
      "content": "(Aplausos)",
      "duration": 14823,
      "startOfParagraph": false,
      "startTime": 475332
    }
```

---

There are likely other instances of this error. These are the 2 we've identified from our logs of failed synchronizations with the Amara API. Our db has a unique key on (translation,startTime), so it's impossible to store these translations as they currently exist.

**Comment by jdragojevic** [comment]
Should be resolved by changes in #1866 deployed 15-Apr 2015.

**Comment by alexdean** [comment]
thanks janet.
